backend/clubs/views.py

```diff
@@ -4950,6 +4961,16 @@ def get_queryset(self):
         )
 
 
+class ApplicationExtensionViewSet(viewsets.ModelViewSet):
+    permission_classes = [ClubSensitiveItemPermission | IsSuperuser]
+    serializer_class = ApplicationExtensionSerializer
+
+    def get_queryset(self):
```

Comment:
Consider using `lookup_field` instead

`get_queryset` defaults to `ApplicationExtension.objects.all()` I believe that is what we want?